src/index.html

```diff
@@ -5,8 +5,422 @@
+              <li class="stars__star stars__star--colored"></li>
+              <li class="stars__star stars__star--colored"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <p>Reviews: 5</p>
+          </article>
+
+          <h3 class="card__price">
+            <p>Price:</p>
+            <p class="card__price-value">$2,199</p>
+          </h3>
+
+          <a
+            href="#buy"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+
+        </li>
+
+        <li
+          class="card catalog__item"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__image"
+          >
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+
+          <article class="card__review">
+            <ul class="stars">
+              <li class="stars__star stars__star--colored"></li>
+              <li class="stars__star stars__star--colored"></li>
+              <li class="stars__star stars__star--colored"></li>
+              <li class="stars__star stars__star--colored"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <p>Reviews: 5</p>
+          </article>
+
+          <h3 class="card__price">
+            <p>Price:</p>
+            <p class="card__price-value">$2,199</p>
+          </h3>
+
+          <a
+            href="#buy"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </li>
+
+        <li
+          class="card catalog__item"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__image"
+          >
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+
+          <article class="card__review">
+            <ul class="stars">
+              <li class="stars__star stars__star--colored"></li>
+              <li class="stars__star stars__star--colored"></li>
+              <li class="stars__star stars__star--colored"></li>
+              <li class="stars__star stars__star--colored"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <p>Reviews: 5</p>
+          </article>
+
+          <h3 class="card__price">
+            <p>Price:</p>
+            <p class="card__price-value">$2,199</p>
+          </h3>
+
+          <a
+            href="#buy"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </li>
+
+        <li
+          class="card catalog__item"
+        >
```

Comment:
as it does not improve readability here - I would leave it as it is
```suggestion
        <li class="card catalog__item">
```